internal/server/middleware/recovery.go

```diff
@@ -8,14 +8,17 @@ import (
 
 	"github.com/go-kratos/kratos/v2/log"
 	"github.com/go-kratos/kratos/v2/middleware/recovery"
+	"go.opentelemetry.io/otel/metric"
 
 	"google.golang.org/grpc"
 )
 
 type Option func(*options)
 
 type options struct { // Duplicated from https://github.com/go-kratos/kratos/blob/main/middleware/recovery/recovery.go b/c no export
-	handler recovery.HandlerFunc
+	handler  recovery.HandlerFunc
```

Comment:
This may be a sign that we should put the middlewares in separate packages the way Kratos does. Dropping them all in one folder wasn't a principled decision- it just didn't seem necessary to split them out when each one was a single file. It makes a difference here, though, because two different middlewares define different options structs.